modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableHeaderRow.java

```diff
@@ -424,15 +448,15 @@ protected NestedTableColumnHeader createRootHeader() {
      *                                                                         *
      **************************************************************************/
 
-    TableColumnHeader getReorderingRegion() {
+    protected TableColumnHeader getReorderingRegion() {
         return reorderingRegion;
     }
 
     void setReorderingColumn(TableColumnBase rc) {
         dragHeaderLabel.setText(rc == null ? "" : rc.getText());
     }
 
-    void setReorderingRegion(TableColumnHeader reorderingRegion) {
+    protected void setReorderingRegion(TableColumnHeader reorderingRegion) {
```

Comment:
I am personally not using the reorderingRegion property. Only the reorderingProperty (see above) is useful to me when a reorder is taking place. I'll let @jperedadnr explain the needs.

Reply:
Sorry, I missed to reply to this comment.
About the region, it makes sense having direct access to it. For instance, for TableView2 we [override](https://bitbucket.org/controlsfx/controlsfx/src/09355e6b74836a8e2281fc4591e449ab36ce30fa/controlsfx/src/main/java/impl/org/controlsfx/tableview2/TableHeaderRow2.java?at=default#TableHeaderRow2.java-262) the setter to adjust the `dragHeader` region when reordering occurs.